content/html/reference/elements/abbr/docs.md

```diff
@@ -1,3 +1,24 @@
+---
+title: '<abbr>: The Abbreviation element'
+mdn_url: https://developer.mozilla.org/docs/Web/HTML/Element/abbr
+tags:
+    group: Inline text semantics
+api: HTMLElement
+permitted_aria_roles:
+    - any
+tag_omission: none
```

Comment:
What are all these (`tags`, `api`, `permitted_aria_roles`, `tag_omission`)??

By the way, in the new read-only MDN site (e.g. beta.developer.mozilla.org) we have removed all display of tags.

Reply:
`api`, `permitted_aria_roles`, `tag_omission` are all things that live in the "technical summary" box (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#Technical_summary).

We didn't think too hard about these when we added them to the recipe, and I'm not sure if we really want all of them to be represented as data (yes on `api` I think, and perhaps `permitted_aria_roles`, but `tag_omission` is tricky to represent structured). Also I think we might want a better name than `api` - it's called "DOM Interface" in the page.

`tags` is just generic tags. We don't have a use for tags yet but I expect we might in future.

> By the way, in the new read-only MDN site (e.g. beta.developer.mozilla.org) we have removed all display of tags.

\o/. Tags in Kuma are a [horrible mess](https://developer.mozilla.org/en-US/docs/tags).

Reply:
I suppose we, like with examples, make that a `tech_summary.yaml` file in the `html/reference/element/video/` directory and have a React "ingredient" component to render it out. But not a blocker for progress.